jscripts/bbcodes_sceditor.js

```diff
@@ -227,7 +227,7 @@ $(function ($) {
 				$cite.remove();
 
 				content = this.elementToBbcode(element);
-				author = '=' + author.replace(/(^\s+|\s+$)/g, '');
+				author = '=' + author.replace(/(^\s+|\s+$)/g,);
```

Comment:
That's wont work. First of all now it will replace all catched chars to 'undefined', cuz you don't provide replacement as second argument. Secondly, that regexp matching only whitespaces at begining and end of the word. There is no replecement for apostrophe. I think, that this is wrong place for fix.

Reply:
Strange, it works fine as a fix on my local host, I guess the , should be removed before merging.

Reply:
There's still issue when you quoting by pressing "reply" button to post. Replying through "quote" seems to work.
![image](https://user-images.githubusercontent.com/4405492/82146661-2e4dab80-984c-11ea-88bd-d09082b83cde.png)

Reply:
And the issue with undefined when you add space before/after username in quote 
![image](https://user-images.githubusercontent.com/4405492/82146997-0874d680-984d-11ea-8353-46f978908b85.png)

Reply:
@Matslom Thanks. Removing the second arguement solves the undefined issue. Still looking into the issue with the reply button.

Reply:
But now, it will not remove whitespaces on beginning and end. You should revert your entire changes.

Reply:
@Matslom Do you plan on submitting a new PR?

Reply:
@Matslom - please check the latest changes :)

Reply:
@Eldenroot Removing whitespaces is ok now. I didn't checked if escaped apostrophe is fixed.